### PR TITLE
fix(eslint-plugin) replace interface-name-prefix

### PR DESCRIPTION
**Pull request by zy2ba**
In typescript-eslint 3.0.0 interface-name-prefix was replaced by naming-convention ([see release notes](https://github.com/typescript-eslint/typescript-eslint/releases/tag/v3.0.0))
Implementation for replacement is taken from naming-convention examples: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/naming-convention.md#enforce-that-interface-names-do-not-begin-with-an-i

### CODE REVIEW

**Review by SuperOleg39 — CHANGES_REQUESTED** [review]

File: packages/linters/eslint/common/typescript/conventions.js [hunk 1]
```diff
@@ -14,9 +14,16 @@ module.exports = tsOnly({
...
         ],
-        '@typescript-eslint/interface-name-prefix': [
+        '@typescript-eslint/naming-convention': [
             breaking({since: 2}),
-            {prefixWithI: 'never'},
+            {
+                selector: 'interface',
+                format: ['PascalCase'],
+                custom: {
+                    regex: '^I[A-Z]',
```
[the remark this answers is not in the document]
Comment (zy2ba) [reply packages/linters/eslint/common/typescript/conventions.js:23]: I was go to last 2.x.x version for search correct equivalent for interface-name-prefix with prefixWithI: 'never' option.
Now i upgraded regexp by fully correct regexp.
[you can see this there](https://github.com/typescript-eslint/typescript-eslint/blob/f18890166146d8c6b8804ef705c04b15da269926/packages/eslint-plugin/src/rules/interface-name-prefix.ts)

del separate naming-convention from `C:\Projects\linters\packages\linters\eslint\common\typescript\conventions.js` and paste this to `C:\Projects\linters\packages\linters\eslint\common\conventions.js`  naming-convention rule

[the remark this answers is not in the document]
Comment (zy2ba) [reply packages/linters/eslint/common/typescript/conventions.js]: I was go to last 2.x.x version for search correct equivalent for interface-name-prefix with prefixWithI: 'never' option.
Now i upgraded regexp by fully correct regexp.
[you can see this there](https://github.com/typescript-eslint/typescript-eslint/blob/f18890166146d8c6b8804ef705c04b15da269926/packages/eslint-plugin/src/rules/interface-name-prefix.ts)

del separate naming-convention from `C:\Projects\linters\packages\linters\eslint\common\typescript\conventions.js` and paste this to `C:\Projects\linters\packages\linters\eslint\common\conventions.js`  naming-convention rule

[the remark this answers is not in the document]
Comment (SuperOleg39) [reply packages/linters/eslint/common/typescript/conventions.js]: Good, looks like current behaviour

**Review by SuperOleg39 — APPROVED** [review]

**Review by MarsiBarsi — APPROVED** [review]

**Comment by SuperOleg39** [comment]
Released at `1.0.0-rc.12`

**Pull request merged**